keg_auth/forms.py

```diff
@@ -51,7 +53,10 @@ class SetPassword(Form):
 
 def get_permission_options():
     perm_cls = flask.current_app.auth_manager.entity_registry.permission_cls
-    return [(str(perm.id), perm.description) for perm in perm_cls.query.order_by('description')]
+    query = perm_cls.query.add_columns(
```

Comment:
Can we do this instead?

```python
return perm_cls.query.with_entities(
    perm_cls.id, coalesce(perm_cls.description, perm_cls.token).label('desc')
).order_by('desc').all()
```
I know performance isn't an issue here, and it used a list comprehension originally, but I think it's a good habit to let SQL do as much of the work as possible while avoiding redundant looping.